example.py

```diff
@@ -64,7 +69,10 @@ def holaMundo(name='Mysterious Someone'):
                     id.search()
                     del id
         except Exception:
+            #try: TODO: antes de romper intentar generar el CSV
```

Comment:
Con respecto a este to-do, me cuesta visualizar cómo generar el .csv por fuera de la clase porque actualmente esa información es privada. Creo que `id` queda fuera del contexto del except y no se puede recuperar.

Reply:
Es verdad, ese no parece ser el mejor lugar para enmedar el bug. Tengo otras ideas que puedo ensayar y luego ponemos en común si es una buena solución.